image/preseed/preseed_linux.go

```diff
@@ -242,7 +242,25 @@ func prepareCore20Mountpoints(opts *preseedCoreOptions) (cleanupMounts func(), e
 		}
 	}
 
+	underWritable := func(path string) string {
+		return filepath.Join(opts.WritableDir, path)
+	}
+
+	// because of the way snapd snap is built, we need the
+	// 'current' symlink to exist when the snapd binary is
+	// invoked, so that any runtime libraries will be correctly
+	// resolved, so bind-mount a snapd snap at a side location set
+	// up 'current' as symlink to it, which can then be easily
+	// replaced by snapd as it preseeds the image
```

Comment:
Same thing about comment being a bit confusing in the description. Also, this comment should probably go a bit down to where the symlink is really created and maybe here say that needs definition now for the clean-up function.

Reply:
Done.